playwright/pages/staffMembersPage.ts

```diff
@@ -30,6 +34,12 @@ export class StaffMembersPage extends BasePage {
     this.inviteStaffMembersDialog = new InviteStaffMembersDialog(page);
   }
 
+  async goToStaffDetailsPage(id: string) {
+    await this.waitForNetworkIdleAfterAction(async () => {
+      await this.page.goto(URL_LIST.staffMembers + id);
+    });
+  }
```

Comment:
questin: why do you wait for network response before entering details page

Reply:
Good point, I will change it